torch/utils/cpp_extension.py

```diff
@@ -171,11 +171,11 @@ def check_compiler_abi_compatibility(compiler):
         if sys.platform == 'linux':
             minimum_required_version = MINIMUM_GCC_VERSION
             version = subprocess.check_output([compiler, '-dumpfullversion', '-dumpversion'])
-            version = version.split('.')
+            version = version.decode().strip().split('.')
```

Comment:
this is not python3 compatble right?
`AttributeError: 'str' object has no attribute 'decode'`

Reply:
nevermind, subprocess returns `bytes` and not `str`.